containers/glassfish/jersey-gf-ejb/src/main/java/org/glassfish/jersey/gf/ejb/internal/EjbComponentProvider.java

```diff
@@ -347,9 +370,23 @@ private void logLookupException(final Method method, final Class<?> component, C
                 allLocalOrRemoteIfaces.add(i);
             }
         }
+        if (allLocalOrRemoteIfaces.isEmpty()) {
+            for (Class<?> i : resourceClass.getInterfaces()) {
+                if (isAcceptableLocalInterface(i)) {
+                    allLocalOrRemoteIfaces.add(i);
+                }
+            }
+        }
         return allLocalOrRemoteIfaces;
     }
 
+    private static boolean isAcceptableLocalInterface(final Class<?> iface) {
+        if ("javax.ejb.".equals(iface.getPackage().getName())) {
```

Comment:
`iface.getPackage().getName()` should be compared with `javax.ejb`

Reply:
In this case it's the same, but general rule is to use constant's equals method because that is guaranteed to be null-safe. SpotBugs has rules for that ;)

Reply:
Ha! You're right! The package name is without the dot at the end! :D
Thank you!